scripts/update-tf-compatibility-matrix.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
```

Comment:
Another big shell script 😅 Did you consider using python?

Reply:
or Go as Agustin did for the one in CFN listing versions in regions ;-)

a great topic you may want to bring to next tech meeting

Reply:
yeah I essentially re-used the existing script

Reply:
or Go as @AgustinBettati  did for the one in CFN listing CFN versions in regions ;-) 
Go templates are nice.

a great topic you may want to bring to next tech meeting